adagrams/game.py

```diff
@@ -1,11 +1,131 @@
+import random
+import array
```

Comment:
Not sure why this library is being imported (I didn't see it used below). It's a good idea to read over a PR before submitting  to catch extraneous items like this one!